Recipes/Apply_function_to_every_gridpoint.ipynb

```diff
@@ -37,12 +37,13 @@
   },
```

Comment:
<p>I assume this long walltime is just something out of our control? Not it actually always being slow ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/461/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>I think what's happening is that things are getting 'gummed' up after the data load, when the system is trying to split out the two statistics and actually compute the merges. I'm going to see what happens if a don't explicitly <code>load</code> the <code>stats</code> array at this point, and let it be implicitly loaded in the plotting step instead.  </p>

Reply:
<p>I just ran the old CC version of the notebook, this step is just as slow there - it gets hung up on the <code>vectorize</code> and <code>transpose</code> processes, regardless of using CC or <code>intake</code> (note that I'm only running on 7 cores).   </p>

Reply:
Ok thanks

Reply:
Can we use a different experiment? I'm hitting this slow loading sst data times. It's a tutorial so whether it's that cycle6 or cycle4 or something else it shouldn't matter...!

Reply:
Well we can leave this for some other time..